### PR TITLE
Update Neptune Example

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
Add README for Neptune example and update the example with the docker setting to run on remote orchestrators

## Pre-requisites
Please ensure you have done the following:
- [X] I have read the **CONTRIBUTING.md** document.
- [X] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: examples/neptune_tracking/pipelines/training_pipeline/training_pipeline.py [hunk 1]
```diff
@@ -12,11 +12,19 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
+from zenml.config import DockerSettings
+from zenml.integrations.constants import NEPTUNE, SKLEARN, TENSORFLOW
 from zenml.pipelines import pipeline
 
+docker_settings = DockerSettings(
+    required_integrations=[TENSORFLOW, SKLEARN, NEPTUNE],
```
Comment (schustmi) [inline examples/neptune_tracking/pipelines/training_pipeline/training_pipeline.py:20]: I couldn't see any mentions of sklearn in the example, do you remember why that was necessary?

Also maybe as an additional point, I would also remove neptune from this list of required integrations as ZenML already detects the integration of all stack components and installs them

File: examples/neptune_tracking/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+### Pre-requisites
+In order to run this example, you need to install and initialize ZenML:
+
+```shell
+# install CLI
+pip install "zenml[server]"
+
+# install ZenML integrations
+zenml integration install tensorflow sklearn neptune
+pip install neptune-client[tensorflow-keras]
```
Comment (schustmi) [inline examples/neptune_tracking/README.md:40]: I think this line would be needed in the `setup.sh` file as well

File: examples/neptune_tracking/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+### Pre-requisites
+In order to run this example, you need to install and initialize ZenML:
+
+```shell
+# install CLI
+pip install "zenml[server]"
+
+# install ZenML integrations
+zenml integration install tensorflow sklearn neptune
```
Comment (schustmi) [inline examples/neptune_tracking/README.md:39]: (Maybe we can remove sklearn here as well? If we do need it, it also needs to be included in the `setup.sh`

File: examples/neptune_tracking/README.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+We will run two experiments with different parameters (epochs and learning rate)
+and log these experiments into a neptune platform. 
+
+In the example script, the [neptune-tensorflow-keras](https://docs.neptune.ai/integrations/keras/) is
+used within the training step to directly hook into the TensorFlow training.  
+It will log out all relevant parameters, metrics and output files Automatically. . Additionally, we explicitly log the test accuracy within the evaluation step.
+
+Note that despite `neptune` being used in different steps within a pipeline, ZenML handles initializing `neptune` and ensures the experiment name is the same as the pipeline name, and the experiment run is the same name as the pipeline run name. This establishes a lineage between pipelines in ZenML and experiments in `neptune`.
+
+# 🖥 Run it locally
```
Comment (schustmi) [inline examples/neptune_tracking/README.md:24]: I think in other examples we also include the "super quick run" using `zenml example run`, I think we should add that here as well

**Review by schustmi — CHANGES_REQUESTED** [review]

File: examples/neptune_tracking/README.md (hunk 4 above)
Comment (safoinme) [reply examples/neptune_tracking/README.md:24]: Wouldn't super quick run need stack component configuration to connect to their platform?

File: examples/neptune_tracking/pipelines/training_pipeline/training_pipeline.py (hunk 1 above)
Comment (safoinme) [reply examples/neptune_tracking/pipelines/training_pipeline/training_pipeline.py:20]: Yes sklearn is not needed, will remove it from list of required integrtions.

File: examples/neptune_tracking/README.md (hunk 4 above)
Comment (schustmi) [reply examples/neptune_tracking/README.md:24]: Ah you're absolutely right, Neptune only has the remote server right? In that case we should delete the `setup.sh` file in order to raise the correct warning/error when trying `zenml example run`

**Review by schustmi — APPROVED** [review]
LGTM

File: examples/neptune_tracking/README.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+# initialize
+zenml init
+
+# Start the ZenServer to enable dashboard access
+zenml up
+```
+
+### 🥞 Create the stack with the Neptune experiment tracker component
+
+In order to use an experiment tracking tool like Neptune, you need to create a new `StackCoomponent`,  and
```
Comment (bcdurak) [inline examples/neptune_tracking/README.md:55]:
```suggestion
In order to use an experiment tracking tool like Neptune, you need to create a new `StackComponent`,  and 
```

**Review by bcdurak — APPROVED** [review]
Aside from a small typo, everything looks good!

**Pull request merged**